uitest/Dialogs/AddWidgetDialog.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using DevHome.UITest.Extensions;
+using DevHome.UITest.Pages;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+
+namespace DevHome.UITest.Dialogs;
+
+/// <summary>
+/// Dialog model class for the add widget dialog
+/// </summary>
+public class AddWidgetDialog : PageDialog<DashboardPage>
+{
+    private WindowsElement SSHNavigationItem => Driver.FindElementByAccessibilityId($"{Configuration.Widget.IdPrefix}!App!!{Configuration.Widget.Provider}!!SSH_Wallet");
+
+    private WindowsElement MemoryNavigationItem => Driver.FindElementByAccessibilityId($"{Configuration.Widget.IdPrefix}!App!!{Configuration.Widget.Provider}!!System_Memory");
+
+    private WindowsElement NetworkUsageNavigationItem => Driver.FindElementByAccessibilityId($"{Configuration.Widget.IdPrefix}!App!!{Configuration.Widget.Provider}!!System_NetworkUsage");
+
+    private WindowsElement GPUUsageNavigationItem => Driver.FindElementByAccessibilityId($"{Configuration.Widget.IdPrefix}!App!!{Configuration.Widget.Provider}!!System_GPUUsage");
```

Comment:
`{Configuration.Widget.IdPrefix}!App!!{Configuration.Widget.Provider}!!` seems to be repeated a lot. Could we do something to not repeat it?